src/Scheduler/GenericInvocationExecutor.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Scheduler;
+
+final class GenericInvocationExecutor implements ScheduledInvocationExecutor
```

Comment:
I think I want to rework the whole invocation structure. Instead of going through the shell again, we can build the command, and then pass it to the `ConsoleApplication` without having to boot tempest once more. 

This is something I'll refactor after merge though, as it might involve some bigger changes.

Reply:
Edit: maybe it won't be possible, because we want to run stuff in the background. Ok, I'll just play around with it and see where it leads me